vault/resource_kv_secret_v2_test.go

```diff
@@ -56,8 +59,28 @@ func TestAccKVSecretV2(t *testing.T) {
 				ImportStateVerify: true,
 				ImportStateVerifyIgnore: []string{
 					"data_json", "disable_read",
-					"delete_all_versions", "mount",
-					"name", "cas",
+					"delete_all_versions", "cas",
+				},
+			},
+			{
+				Config: testKVSecretV2Config_initial(updatedMount, updatedName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, consts.FieldMount, updatedMount),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldName, updatedName),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldPath, fmt.Sprintf("%s/data/%s", updatedMount, updatedName)),
+					resource.TestCheckResourceAttr(resourceName, "delete_all_versions", "true"),
+					resource.TestCheckResourceAttr(resourceName, "data.zip", "zap"),
+					resource.TestCheckResourceAttr(resourceName, "data.foo", "bar"),
+					resource.TestCheckResourceAttr(resourceName, "data.flag", "false"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
+					"data_json", "disable_read",
+					"delete_all_versions", "cas",
```

Comment:
are we sure none of these fields can be imported?

Reply:
Just double-checked. Looks like `cas` can be imported! Rest remain in the Verify Ignore list